internal/car/car_test.go

```diff
@@ -85,9 +85,9 @@ usr/local/bin/car
 			veryVerbose: true,
 			patterns:    []string{"usr/local/bin/car"},
 			expectedOut: `fake://ghcr.io/v2/tetratelabs/car/manifests/v1.0 platform=linux/amd64 totalLayerSize: 150
-fake://ghcr.io/v2/tetratelabs/car/blobs/sha256:4e07f3bd88fb4a468d5551c21eb05f625b0efe9ee00ae25d3ffb87c0f563693f size=30
-CreatedBy: /bin/sh -c #(nop) ADD file:d7fa3c26651f9204a5629287a1a9a6e7dc6a0bc6eb499e82c433c0c8f67ff46b in / 
-fake://ghcr.io/v2/tetratelabs/car/blobs/sha256:15a7c58f96c57b941a56cbf1bdd525cdef1773a7671c52b7039047a1941105c2 size=30
+fake://ghcr.io/v2/blobs/sha256:4e07f3bd88fb4a468d5551c21eb05f625b0efe9ee00ae25d3ffb87c0f563693f size=30
```

Comment:
it is ok for now that the test data doesn't include "tetratelabs/car/" in the blob names. Later, if we need a tight relationship between image and its layers (in the code model) we can refactor. For now, the fake is easier to write with global fake layers, which is fine as there's only one image in it anyway.